src/pages/settings/index.jsx

```diff
@@ -18,21 +18,21 @@ export default function Settings() {
 
   const [language, setLanguage] = React.useState(i18n.language);
 
-  const { devMode, setDevMode, visiblePages } =
+  const { devMode, setDevMode, visiblePages, setVisiblePages } =
     React.useContext(DevModeContext);
 
   const handleDevModeChange = (event) => {
     localSettings.update(localSettingsKeys.isDevMode, event.target.checked);
     setDevMode(event.target.checked);
-    Object.keys(visiblePages).forEach((pageUrl)=>{
-      visiblePages[pageUrl]=event.target.checked
-      localSettings.update(localSettingsKeys.visiblePages[pageUrl], event.target.checked);
-    })
```

Comment:
Don't care about that, the pages will just appear as they was before. Not a problem

Reply:
I added this so that when devmode is switched on, regardless of the previous states, both nutrition and insights will be turned on